### ISSUE TITLE
Change repo name to org.gnome.Meld 

### ISSUE THREAD

**Issue by FakeShemp** [flathub/org.gnome.meld#24]
The app seems to have adopted a majuscule M in their name; `org.gnome.Meld`. I think the repo here should be changed as well.

**Comment by kaiw** [comment]
It was changed to match the GNOME application ID style. I'm sorry, I
didn't consider flathub in the fallout when I updated the name.

**Comment by bilelmoussaoui** [comment]
Please follow https://github.com/flathub/flathub/wiki/App-Maintenance#end-of-life to EOL the current app-id & resubmit it again.

**Comment by bilelmoussaoui** [comment]
Ah never mind. No need for that

**Comment by bilelmoussaoui** [comment]
Though the app-id change didn't happen yet, at least not part of a stable release?

**Comment by kaiw** [comment]
No, the app-id change is in the current unstable release. It was done as part of a larger change to get nightly flatpak CI builds working in the GNOME infrastructure, so we now have the "correctly" capitalised name and a `org.gnome.MeldDevel` ID as well.

**Comment by bilelmoussaoui** [comment]
Sure, feel free to ping me once that release is ready so I can update the repository name.

**Comment by hfiguiere** [comment]
See the patches I needed in #37 

One doesn't simply rename a repository

**Comment by bilelmoussaoui** [comment]
> See the patches I needed in #37 
> 
> One doesn't simply rename a repository

Why not? We did that few times already in the past.

**Comment by hfiguiere** [comment]
it cool to have these admin privileges. Barthalion always said "resubmit with a new name and deprecate"

**Comment by bilelmoussaoui** [comment]
I am not talking about admin privileges here. What I mentioned is in this specific case, where the change is a letter going from lowercase -> uppercase, there is no other option other than renaming the repository as you cannot archive it and create a new one that has almost the same name.

**Comment by TheEvilSkeleton** [comment]
FWIW I suggested in [!112](https://gitlab.gnome.org/GNOME/meld/-/merge_requests/112) that we should mark this as EOL and resubmit under the `org.gnome.Meld` ID.

**Comment by bilelmoussaoui** [comment]
You cannot, see my comment above.

**Comment by TheEvilSkeleton** [comment]
Oh, sorry
